templates/repo/home.tmpl

```diff
@@ -28,22 +28,28 @@
 		{{if .IsRepositoryAdmin}}<a id="manage_topic" style="cursor:pointer;margin-left:10px;">{{.i18n.Tr "repo.topic.manage_topics"}}</a>{{end}}
 		</div>
 		{{if .IsRepositoryAdmin}}
-		<div class="ui repo-topic-edit grid" id="topic_edit" >
+		<div class="ui repo-topic-edit grid form segment error" id="topic_edit" >
 			<div class="fourteen wide column">
+				<div class="field">
 				<div class="ui fluid multiple search selection dropdown">
```

Comment:
As there is now a additional div wrapping this div, can you indent?

Reply:
Fixed